scripts/asm-installer/install_asm

```diff
@@ -2229,14 +2229,13 @@ create_istio_namespace() {
   retry 2 kubectl create ns istio-system
 }
 
-exit_if_istio_namespace_not_exists() {
+warn_if_istio_namespace_not_exists() {
   info "Checking for istio-system namespace..."
   if ! istio_namespace_exists; then
-    { read -r -d '' MSG; fatal "${MSG}"; } <<EOF || true
+    { read -r -d '' MSG; warn "${MSG}"; } <<EOF || true
 The istio-system namespace doesn't exist.
-Please create the "istio-namespace" and retry, or run the script with the
-'--enable_namespace_creation' flag to allow the script to enable it on your behalf.
-$(enable_common_message)
+Please create the "istio-namespace", or make sure that the user installing ASM
```

Comment:
What is "istio-namespace"?

Reply:
Oops, thanks.

Reply:
Yep I think there should be "istio-system"